scripts/jaeger.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
```

Comment:
This might be useful to add to the docker-compose file instead of a separate script.

Reply:
moved